### ISSUE TITLE
Payment Gateway is not shown in checkout page

### ISSUE THREAD

**Issue by hariprasad2k0** [iamsayan/upi-qr-code-payment-for-woocommerce#7]
I have added the plugin and merchant id 9003688688-1@okbizaxis

![image](https://user-images.githubusercontent.com/5036661/128594170-5971c062-6dd0-4c58-bbd3-4d5882828402.png)

Product URL : https://akconline.com/product/payment-test/

Screenshot of Active Payment Methods
![image](https://user-images.githubusercontent.com/5036661/128594216-721539a7-e06e-4f16-9d2f-c6e2817e5400.png)

Screenshot of Checkout Page
![image](https://user-images.githubusercontent.com/5036661/128594228-7c3011f0-540a-42aa-a6fd-ff6ad1d14970.png)

Kindly check and do needful, i need your support.

**Comment by TandelK** [comment]
Try to move up Payment Gateway to very first priority and check .

**Comment by hariprasad2k0** [comment]
you are right, the problem is because of checkout page, i was using woocommerce blocks checkout page. When i removed it your plugin works like a charm.

is there any way to make it work with woocommerce block checkout block.
